Library/Formula/libfuse.rb

```diff
@@ -0,0 +1,25 @@
+class Libfuse < Formula
+  desc "Reference implementation of the Linux FUSE (Filesystem in Userspace) interface."
+  homepage "https://github.com/libfuse/libfuse"
+  url "https://github.com/libfuse/libfuse/archive/fuse_2_9_5.tar.gz"
+  sha256 "ccea9c00f7572385e9064bc55b2bfefd8d34de487ba16d9eb09672202b5440ec"
+  head "https://github.com/libfuse/libfuse.git"
+
+  def install
+    system "./makeconf.sh"
+    system "./configure", "--prefix=#{prefix}", "--disable-util"
```

Comment:
It is hardcoded to install into /sbin, for instance /sbin/mount.fuse.  I can patch it to install them to `#{sbin}`.